src/assets/scss/_themes.scss

```diff
@@ -43,6 +43,8 @@ $editionColors: (
         eventMiddle: #ffffb0,
         eventDarker: #fcfc60,
     ),
+    commentNotes: rgb(87, 14, 105),
+    criticalNotes: rgb(24, 70, 155),
```

Comment:
In this file I think it's better to use the variables for the colors of the theme, but we can refactor it in the future.